backend/segmentation/api/mt_metrics.py

```
@@ -205,6 +205,19 @@ class MTMetricsResponse(BaseModel):
+# Re-exported deliberately: `test_mt_metrics_band.py` reaches for these by name
+# to assert the endpoint's helpers ARE the shared objects, so an `__all__` entry
+# states "unused here, used by the contract test" instead of leaving a reader to
+# infer it from an unused-global warning.
+__all__ = [
+    "_polyline_length",
+    "_imagej_median",
+    "_fill_convex_polygon",
+    "_rasterize_band",
+    "_dilate",
+    "_vicinity_mask",
+]
```

Ruff (0.16.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)